src/Proxy.php

```diff
@@ -222,6 +226,47 @@ public function disableAutoRefresh(): self
         return $this;
     }
 
+    public function disableDoctrineEvents(?string $eventTypeName = null): self
```

Comment:
Do you have a use-case for having these methods on the `Proxy`? If not, I'd prefer this code to be in `Factory` (and remove this public API).

Reply:
done :+1: